src/Nethermind/Nethermind.Merge.Plugin/MergePlugin.cs

```diff
@@ -301,6 +302,8 @@ public Task InitRpcModules()
                 _api.LogManager,
                 TimeSpan.FromSeconds(_blocksConfig.SecondsPerSlot));
 
+            _api.RpcCapabilitiesProvider = new EngineRpcCapabilitiesProvider(_api.SpecProvider);
```

Comment:
I'm wondering about startup loading. As far as I remember, RpcModules are loaded after the health checks plugin, so it might lead to issues

Reply:
By default Merge is initialized before HealthChecks
`PluginOrder = { "Clique", "Aura", "Ethash", "AuRaMerge", "Merge", "MEV", "HealthChecks", "Hive" };`

Reply:
```
[RunnerStepDependencies(... typeof(InitializePlugins), ...)]
public class RegisterRpcModules : IStep
```